spec/helpers/application_helper/toolbar_builder_spec.rb

```diff
@@ -1447,12 +1447,23 @@ def method_missing(sym, *args)
       context "and id = miq_template_scan" do
         before { @id = "miq_template_scan" }
 
-        it "and !@record.has_proxy?" do
+        it "returns true for hiding the button when @record neither has proxy nor supports smartstate analysis" do
+          allow(@record).to receive_messages(:supports_smartstate_analysis? => false, :has_proxy? => false)
```

Comment:
@jameswnl Yes, earlier SSA check was not included in tests. The button visibility here depends on two factors, SSA support and has_proxy?. I thought we should have tests that cover all the 4 boolean scenarios for these two variables, hence included 4 tests.